spring-cloud-aws-autoconfigure/src/test/java/io/awspring/cloud/autoconfigure/config/parameterstore/ParameterStoreConfigDataLoaderIntegrationTests.java

```diff
@@ -189,8 +189,8 @@ void serviceSpecificEndpointTakesPrecedenceOverGlobalAwsRegion() {
 				"--spring.cloud.aws.endpoint=http://non-existing-host/",
 				"--spring.cloud.aws.parameterstore.endpoint=" + localstack.getEndpointOverride(SSM).toString(),
 				"--spring.cloud.aws.credentials.access-key=noop", "--spring.cloud.aws.credentials.secret-key=noop",
-				"--spring.cloud.aws.region.static=eu-west-1",
-				"--logging.level.io.awspring.cloud.parameterstore=debug")) {
+				"--spring.cloud.aws.region.static=eu-west-1", "--logging.level.io.awspring.cloud.parameterstore=debug",
+				"--spring.cloud.aws.dynamodb.enabled=false", "--spring.cloud.aws.s3.enabled=false")) {
```

Comment:
Why changes in this file were needed?

Reply:
only dynamodb.enabled=false, reason for this is since we are putting whole application context up it is trying to configure `ClusterDaxClient` since it has dependency. It will fail since `url` property is null and while trying to configure a client nullptr will happen since `url` is mandatory field.
Example:
`Caused by: java.lang.NullPointerException
    at software.amazon.dax.com.amazon.dax.client.HostPort.parse(HostPort.java:111)`